lib/lib-openapi/src/test/java/org/sagebionetworks/translator/ControllerToControllerModelTranslatorTest.java

```diff
@@ -0,0 +1,682 @@
+		// call under test
+		assertEquals(new LinkedHashMap<>(),
+				translator.getParameterToDescription(new ArrayList<>(Arrays.asList(mockReturnComment))));
+	}
+
+	@Test
+	public void testGetParameterToDescriptionWithNullBlockTags() {
+		// call under test
+		assertEquals(new LinkedHashMap<>(), translator.getParameterToDescription(null));
+	}
+
+	@Test
+	public void testGetParameterToDescriptionWithEmptyBlockTags() {
+		// call under test
+		assertEquals(new LinkedHashMap<>(), translator.getParameterToDescription(new ArrayList<>()));
+	}
+
+	@Test
+	public void testGetReturnCommentWithNull() {
+		// call under test
+		assertEquals(Optional.empty(), translator.getReturnComment(null));
+	}
+
+	@Test
+	public void testGetReturnCommentWithEmptyArray() {
+		// call under test
+		assertEquals(Optional.empty(), translator.getReturnComment(new ArrayList<>()));
+	}
+
+	@Test
+	public void testGetReturnCommentWithoutReturnTreeInArray() {
+		ParamTree mockParamComment = Mockito.mock(ParamTree.class);
+		when(mockParamComment.getKind()).thenReturn(DocTree.Kind.PARAM);
+		// call under test
+		assertEquals(Optional.empty(), translator.getReturnComment(new ArrayList<>(Arrays.asList(mockParamComment))));
+	}
+
+	@Test
+	public void testGetReturnCommentWithEmptyDescription() {
+		List<DocTree> blockTags = new ArrayList<>();
+		ReturnTree mockReturnComment = Mockito.mock(ReturnTree.class);
+		when(mockReturnComment.getKind()).thenReturn(DocTree.Kind.RETURN);
+		when(mockReturnComment.getDescription()).thenReturn(new ArrayList<>());
+		blockTags.add(mockReturnComment);
+
+		assertEquals(Optional.empty(), translator.getReturnComment(blockTags));
+	}
+
+	@Test
+	public void testGetReturnComment() {
+		List<DocTree> blockTags = new ArrayList<>();
+		ReturnTree mockReturnComment = Mockito.mock(ReturnTree.class);
+		when(mockReturnComment.getKind()).thenReturn(DocTree.Kind.RETURN);
+		List<? extends DocTree> descriptions = Mockito.mock(List.class);
+		when(descriptions.toString()).thenReturn(METHOD_RETURN_COMMENT);
+		Mockito.doReturn(descriptions).when(mockReturnComment).getDescription();
+		blockTags.add(mockReturnComment);
+
+		// call under test
+		assertEquals(Optional.of(METHOD_RETURN_COMMENT), translator.getReturnComment(blockTags));
+	}
+
+	@Test
+	public void testGetBehaviorCommentWithNull() {
+		// call under test
+		assertEquals(Optional.empty(), translator.getBehaviorComment(null));
+	}
+
+	@Test
+	public void testGetBehaviorCommentWithEmptyArray() {
+		// call under test
+		assertEquals(Optional.empty(), translator.getBehaviorComment(new ArrayList<>()));
+	}
+
+	@Test
+	public void testGetBehaviorComment() {
+		List<? extends DocTree> fullBody = Mockito.mock(List.class);
+		when(fullBody.isEmpty()).thenReturn(false);
+		when(fullBody.toString()).thenReturn(METHOD_BEHAVIOR_COMMENT);
+		// call under test
+		assertEquals(Optional.of(METHOD_BEHAVIOR_COMMENT), translator.getBehaviorComment(fullBody));
+	}
+}
```

Comment:
add verify checks to for the cases where a spy was called (and check the passed parameters if possible).